dokan/dokan.h

```diff
@@ -69,7 +69,7 @@ extern "C" {
 
 /**
  * \defgroup DOKAN_OPTION DOKAN_OPTION
- * \brief All DOKAN_OPTION flags used in DOKAN_OPTIONS.Options
+ * \brief All DOKAN_OPTION flags used in \ref DOKAN_OPTIONS.Options
```

Comment:
For information, doxygen does not always need the \ref. In this case, doxygen already understood it without \ref.